src/navigator-card-body.jsx

```diff
@@ -292,7 +292,45 @@ export const NavigatorCardBody = ({
         }
     };
 
-    const handleClick = (ev, file) => {
+    const getFilenameForEvent = event => {
+        let data_item = null;
+        let elem = event.target;
+        // Limit iterating over all parents
+        for (let i = 0; i < 10; i++) {
+            data_item = elem.getAttribute("data-item");
+            if (data_item)
+                break;
+
+            if (elem.parentElement)
+                elem = elem.parentElement;
+            else
+                break;
+        }
+
+        return data_item;
+    };
+
+    const handleDoubleClick = (ev) => {
+        const name = getFilenameForEvent(ev);
+        const file = sortedFiles?.find(file => file.name === name);
+        if (!file)
+            return null;
+        if (!file) {
+            resetSelected(ev);
+            return;
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-249-20240305-140311-42a1f07c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)